modules/serverless-custom-revision-urls.adoc

```diff
@@ -0,0 +1,37 @@
+[id="serverless-custom-revision-urls_{context}"]
+= Custom URLs for revisions
+
+Assigning a `--tag` flag to a service by using the `kn service update` command creates a custom URL for the revision that is created when you update the service. The custom URL follows the pattern `http(s)://<tag>-<service_name>-<namespace>.<domain>`.
+
+The `--tag` and `--untag` flags use the following syntax:
+
+* Require one value.
+* Denote a unique tag in the traffic block of the service.
+* Can be specified multiple times in one command.
+
+[id="serverless-custom-revision-urls-assign"]
```

Comment:
For all the ID attributes in this file, please add `_{context}` according to the doc guidelines for [anchoring in modules files](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#anchoring-in-module-files).